SharpCompress.sln

```diff
@@ -1,44 +1,44 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
So revert it then please.  Whitespace changes make a lot of noise for reviewing.